ci/cloudbuild/build.sh

```diff
@@ -250,6 +250,7 @@ if [[ "${DOCKER_FLAG}" = "true" ]]; then
     "--env=USER=$(id -un)"
     "--env=TERM=$([[ -t 0 ]] && echo "${TERM:-dumb}" || echo dumb)"
     "--env=TZ=UTC0"
+    "--env=CI_LIB_IO_FIRST_TIMESTAMP=${CI_LIB_IO_FIRST_TIMESTAMP:-}"
```

Comment:
Is this just to enable manual testing of the code, or is it needed for something in the normal functioning of this feature?

Reply:
I believe it is needed to get the original "first timestamp" through the `docker run` from one `io.sh` environment to a child.

Reply:
I think you're right, that _would_ be needed, but I don't believe any of our builds do that. None of our builds re-invoke `ci/cloudbuild/build.sh`. If that's all it's for, I suspect we don't need to pass along that environment variable.

Reply:
It happens for me all the time.  Say ...
```
ci/cloudbuild/build.sh --docker --trigger=asan-pr
  docker run ... --env=...  ... ci/cloudbuild/build.sh --local asan
    ci/cloudbuild/build.sh --local asan
```

Reply:
I see what you want now. I was thinking of something slightly different. But now that I understand, I think what you have here looks great. Thanks. LGTM.